### PR TITLE
Update workflow to include stepactions

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

File: .github/workflows/generate-experimental-catalogs.yaml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
         run: |
           set -Eeu
           mkdir -p experimental/tasks experimental/pipelines
+          mkdir -p experimental/tasks experimental/stepactions
```
Comment (vdemeester) [inline .github/workflows/generate-experimental-catalogs.yaml:44]: can be all on one line 👼🏼

**Review by vdemeester — COMMENTED** [review]
one nit

File: .github/workflows/generate-experimental-catalogs.yaml (hunk 1 above)
Comment (savitaashture) [reply .github/workflows/generate-experimental-catalogs.yaml]: ah sure
Updated

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**